libvmi/performance.c

```diff
@@ -53,52 +53,6 @@ print_measurement(
            (long int) ktv_end.tv_usec, *diff);
 }
 
-static double
```

Comment:
They break compiling with -Werror so if we want to keep them we can just comment them out.